Classes/Systems/ShortcutHandler.swift

```diff
@@ -27,10 +27,10 @@ struct ShortcutHandler {
         navigationManager: RootNavigationManager
         ) -> Bool {
         switch route {
-        case .tab(let tab):
+        case let .tab(tab):
```

Comment:
Why change this format? We follow the previous style throughout the rest of the app

Reply:
Got it 👍